src/components/courseTimeline.tsx

```diff
@@ -0,0 +1,38 @@
+const CourseTimeline = () => {
+  return (
+    <div className="flex w-3/4 flex-col items-center rounded-xl bg-[#023851] px-[6vw] py-[6vh] font-malayalam text-[#FFF4B8]">
```

Comment:
remove font-malayalam and make sure the colors you're using here are from the tailwind config -- ex: instead of bg-[#023851] you can straight up use bg-devstation-blue-100

Reply:
make sure you solve your merge conflict as well